pkg/httputil/httputil_test.go

```diff
@@ -70,25 +70,55 @@ func TestHttputilNewClient(t *testing.T) {
 	require.Equal(t, httputilServerMsg, string(body))
 }
 
+func TestStatusCodeCreated(t *testing.T) {
+	t.Parallel()
+
+	port := 8305
```

Comment:
It seems we can use `httptest.NewServer` to avoid port conflict.

Reply:
addressed.